lib/git_hooks/installer.rb

```diff
@@ -13,7 +14,7 @@ def install(force = false)
       hook_script = hook_template
       hook_script.gsub!('/usr/bin/env ruby', ruby_path) if ruby_path
 
-      puts "Writing to file #{hook_path}"
+      logger.info "Writing to file #{hook_path}" if logger
```

Comment:
Oh woops, I thought that the test environment wouldn't log `INFO` messages, but that's not the case. I think you're idea is better (use a stub logger for tests)